### PR TITLE
add control key modal created

### LINKED ISSUE #88: `Add Control Key`
blocked by #116 

Image

## Acceptance Criteria
- Clicking outside of the box closes the modal

### PR DESCRIPTION
**Pull request by claireolmstead**
closes #88

### CODE REVIEW

File: src/components/AddControlKey.svelte [hunk 1]
```diff
@@ -91,31 +90,25 @@
...
-  </form>
-</div>
-<TransactionStatus bind:showSelf={showTransactionStatus} statuses={txnStatuses} />
+<Modal id="add-control-key" {isOpen} {close}>
+  <h2 class="section-title-underlined">{`Add Control Key to MSA (${$storeMsaInfo?.msaId})`}</h2>
+
+  <DropDownMenu
+    id="AddControlKey"
+    label="Public Key"
+    selected={$transactionSigningAddress}
```
Comment (rlaferla) [inline src/components/AddControlKey.svelte:99]: `transactionSigningAddress` is obsolete.  Use `$user.signingKey` instead.  Also, do you need to bind the prop?

File: src/components/AddControlKey.svelte [hunk 2]
```diff
@@ -45,31 +40,34 @@
...
-    let endpointURI: string = thisDotApi.selectedEndpoint || '';
-    if (selectedKeyToAdd === '') {
+    let endpointURI: string = $dotApi.selectedEndpoint || '';
+    if (!selectedKeyToAdd) {
       alert('Please choose a key to add.');
     } else {
-      let newKeys = validAccounts[selectedKeyToAdd];
-      let signingKeys = validAccounts[signingAddress];
+      let newKeys = selectedKeyToAdd;
+      let signingKeys = $transactionSigningAddress;
```
Comment (rlaferla) [inline src/components/AddControlKey.svelte:48]: `transactionSigningAddress` is obsolete.

File: src/components/AddControlKey.svelte [hunk 3]
```diff
@@ -1,25 +1,29 @@
 <script lang="ts">
-  import { dotApi, storeConnected, storeCurrentAction, transactionSigningAddress } from '$lib/stores';
+  import { dotApi, transactionSigningAddress, storeMsaInfo } from '$lib/stores';
```
Comment (rlaferla) [inline src/components/AddControlKey.svelte:2]: `transactionSigningAddress` is obsolete.

**Review by rlaferla — CHANGES_REQUESTED** [review]

**Comment by claireolmstead** [comment]
closed in favor of #146

**Pull request closed without merging**